### PR TITLE
Chris/create join group

### PR DESCRIPTION
**Pull request by ctlchan**
Implemented the Join page. Made it so that you need a `group_id` in the URL that is valid, or else you are redirected to the home page; did the same for the preferences page. Also added a default route just so non-existent routes can redirect. 

I wanted to also make the Home page redirect to preferences after successfully creating a group, but I will address that in a later commit to the branch (the `group_id` should be in the response I think?)

### CODE REVIEW

**Comment by ctlchan** [comment]
Implemented redirect to preferences after successfully creating a group

File: web/meetup-facilitator/src/App.jsx [hunk 1]
```diff
@@ -16,7 +17,10 @@ function App() {
         <Routes>
           <Route path='/' element={<Home/>} />
           <Route path='/login' element={<Login/>} />
-          <Route path='/preferences' element={<Preferences/>}/>
+          <Route path='/preferences/:group_id' element={<Preferences/>}/>
+          <Route path='join/:group_id' element={<Join/>}/>
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/App.jsx]: Let's start `join` with a `/` to keep it consistent.

File: web/meetup-facilitator/src/pages/Home.jsx [hunk 2]
```diff
@@ -66,7 +66,12 @@ function Home() {
                 'Content-Type': 'application/json'
             }
         })
-        .then((response) => console.log(response.json()))
+        .then((response) => (response.json()))
+        .then( (json) => { 
+
+            if (Object.keys(json).length > 1)
```
Comment (pabloj2001) [inline web/meetup-facilitator/src/pages/Home.jsx]: Let's check that `json` explicitly contains `group_id`

**Review by pabloj2001 — CHANGES_REQUESTED** [review]
Good job! Only a few minor changes from me

**Review by pabloj2001 — APPROVED** [review]
LGTM!

**Review by marissa-anj — APPROVED** [review]

**Pull request merged**